Dockunit.json

```diff
@@ -0,0 +1,36 @@
+{
+  "containers": [
+    {
+      "prettyName": "nodejs latest",
+      "image": "node:latest",
+      "testCommand": "npm test",
+      "beforeScripts": [
+        "npm install"
+      ]
+    },
+    {
+      "prettyName": "nodejs 5.0.0",
+      "image": "node:5.0.0",
+      "testCommand": "npm test",
+      "beforeScripts": [
+        "npm install"
+      ]
+    },
+    {
+      "prettyName": "nodejs 4.2.1",
+      "image": "dockunit/prebuilt-images:nodejs-mongodb-mocha-jasmine-4.2.1",
```

Comment:
Those images just have that stuff bundled. I'll push something with lighter images.